app/controllers/samples_controller.rb

```diff
@@ -223,36 +223,18 @@ def heatmap
         Viruses: ["Phage"]
       },
       metrics: [
+        "NT.r",
         "NT.aggregatescore",
         "NT.rpm",
-        "NT.r",
         "NT.zscore",
         "NT.maxzscore",
-        "NR.aggregatescore",
         "NR.rpm",
         "NR.r",
-        "NR.zscore",
-        "NR.maxzscore"
+        "NR.zscore"
```

Comment:
Why did you remove NR.maxzscore?

Reply:
because it is the same as NT.maxzscore. `maxzscore == max(NT.zscore, NR.zscore)`

Reply:
I see. It confused me because here still has the NT prefix. Thanks